app/models/client.rb

```diff
@@ -183,6 +183,10 @@ def self.locale_counts
     result
   end
 
+  def unlock_for_login!
+    unlock_access! if locked_at.present? && !access_locked?
```

Comment:
i added `locked_at.present?` check here, b/c without that, the record would unlock every time we call `unlock_for_login!`
`access_locked?` is false if `locked_at` is `nil`, so just using that check wasn't sufficient

Reply:
this never works for clients, b/c `lock_expired?` does not evaluate to `true` ever (strategy is not set)